spec/praxis/trait_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe Praxis::Trait do
+
+  subject(:trait) do
+    Praxis::Trait.new do
+      description 'my awesome trait'
+
+      routing do
+        prefix '/:app_name'
+      end
+
+      response :something
+      response :nothing
+
+      params do
+        attribute :app_name, String
+        attribute :order, String,
+          description: "Field to sort by."
+      end
+
+    end
+  end
+
+  context 'describe' do
+    subject(:describe) { trait.describe }
+
+    its([:description]) { should eq('my awesome trait') }
+    its([:responses]) { should eq [[:something,{}], [:nothing,{}]] }
```

Comment:
my guess is that this shouldn't happen.
If it does we should:
* override it with the new one...(and warn about the situation).
* or simply blow up saying that this cannot happen.

I'm tempted to do second option as it is simpler and more restrictive, and still could open the door to number 1 in the future if we wanted to.